applications/brain_dataset_tutorial.md

```diff
@@ -0,0 +1,77 @@
+# Using Dask and Napari to view and process 100µm resolution human brain dataset.
+We used napari to view and process 100µm resolution human brain large dataset, and in the following tutorial we will break down how it all builds up!
+
+The dataset was obtained from Dr. Brian Edlow's Lab for NeuroImaging of Coma and Consciousness, which is dedicated to promoting recovery of consciousness in people with severe brain injuries.
+This dataset is a 100 micron resolution magnetic resonance imaging (MRI) scan of an ex vivo human brain specimen. The brain specimen was donated by a 58-year-old woman who had no history of neurological disease and died of non-neurological causes.
+
+A very important concept to bear in mind when going through this tutorial is the [Lazy Loading](https://en.wikipedia.org/wiki/Lazy_evaluation), i.e., only loading images to memory when the position slider on napari maps to that particular image, which is very of use, when using large datasets.
```

Comment:
I think this comment about the lazy loading should go after the getting the data in the data loading section